hs_core/tests/api/rest/test_move_or_rename_file_or_folder.py

```diff
@@ -53,27 +53,27 @@ def test_bad_requests(self):
         unzip_url = "/hsapi/resource/%s/functions/move-or-rename/" % self.pid
         response = self.client.post(unzip_url, {
             "source_path": " ",
-            "target_path": "foo"
+            "target_path": "data/contents/foo"

```

Comment:
@pkdash I think it should, but it doesn't, and I am unwilling to change it in this PR. Pre-existing condition. Reason: user might want to create bag-like structures outside data/contents. This test was writing outside data/contents before!

I am willing to coordinate fixing this with you. But I'd rather do it in a later PR. 

For now I have put guard clauses in the script so that it can't write outside data/contents. It shouldn't have to. 

This is just one example of the many kinds of potential file corruption I have found.